### PR TITLE
Restructure code

### PR DESCRIPTION
**Pull request by vietanhdev**
Refactor to new structure.

### CODE REVIEW

File: include/daisykitsdk/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    + **types** (daisykit::types): data structures and types
+    + **visualizers** (daisykit::visualizers): visualizers for object detection, segmentation, keypoints...
+    + **profilers** (daisykit::profilers): profilers to tracking system performance 
+      + node_profiler.h
+      + fps_calculator.h
+      + latency_tracking.h
+      ...
+    + **utils** (daisykit::utils): common utilities
+      + timer.h
+      + filesystem.h
```
Comment (MDuc-ai) [inline include/daisykitsdk/README.md:17]: filesystem logic should be put in io

File: include/daisykitsdk/common/types/keypoint.h [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+#include <opencv2/opencv.hpp>
+
+namespace daisykit {
+namespace types {
+
+struct Keypoint {
+  float x;
+  float y;
+  float prob;  /// Score of this point
```
Comment (MDuc-ai) [inline include/daisykitsdk/common/types/keypoint.h]:
```suggestion
  float prob;  /// Probability of existence of this point
```

File: src/examples/demo_face_detector.cpp [hunk 3]
```diff
@@ -25,7 +25,7 @@
 using namespace cv;
 using namespace std;
 using json = nlohmann::json;
-using namespace daisykit::common;
+using namespace daisykit::types;
 using namespace daisykit::flows;
```
Comment (MDuc-ai) [inline src/examples/demo_face_detector.cpp]:
```suggestion
namespace daisykit {
```

Comment (MDuc-ai) [reply src/examples/demo_face_detector.cpp]: We should use namespace daisykit and add names of submodule before their corresponding declarations.

Comment (MDuc-ai) [reply src/examples/demo_face_detector.cpp]: Apply for all

**Review by MDuc-ai — CHANGES_REQUESTED** [review]
- pushup_analyzer stored in processor seems not a great idea. Maybe a submodule others is better.
- filesystem should be put in submodule io
- Add new line to the end of files.

File: include/daisykitsdk/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      ...
+  + **processors** (daisykit::processors) signal/image processors, basic operations to build computational system.
+    + **image_processors**
+    + **signal_processors**
+    + **trackers**
```
Comment (congphase) [inline include/daisykitsdk/README.md:22]: The difference between ` trackers ` and ` image/signal processors ` is not clear. For example, with ` trackers ` , we track objects on images, right? So `image_processors` deal with images, and `trackers` do the same. Therefore, considering ` trackers ` and ` image_processors ` as individual elements of `processors` seems ambiguous.

File: include/daisykitsdk/README.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  + **graphs** (daisykit::graphs): graph API
+    + **core**: core definitions of graph API
+      + packet.h
+      + node.h
+      ...
+    + **nodes** (daisykit::graphs::nodes): computational or visualizer nodes.
+      + **models**
+      + **image_processors**
+      + **signal_processors**
+      + **trackers**
```
Comment (congphase) [inline include/daisykitsdk/README.md:33]: The same as my previous comment.

**Review by congphase — COMMENTED** [review]
Comment on the structure of `processors` in `include/daisykitsdk/README.md`.

**Pull request merged**